docs/intro/intro_samples.rst

```diff
@@ -0,0 +1,45 @@
+.. _intro_samples:
+
+=======
+Samples
+=======
+
+dimod provides a :class:`~dimod.SampleSet` class that contains, and enables you to
+manipulate, samples.
+
+For an introduction to returned solutions and samples, see
+:std:doc:`Solutions <oceandocs:concepts/solutions>`.
+
+For all supported sampleset methods, see :ref:`sampleset`.
+
+This example creates a sampleset then demonstrates some :class:`~dimod.SampleSet`
```

Comment:
"creates a sampleset and then"

Reply:
"creates a sample set and then" <- notice the space.